src/main/scala/net/ceedubs/ficus/FicusConfig.scala

```diff
@@ -9,6 +9,8 @@ trait FicusConfig {
 
   def as[A](path: String)(implicit reader: ValueReader[A]): A = reader.read(config, path)
 
+  def as[A](implicit reader: ValueReader[A]): A = reader.read(config, ".")
```

Comment:
this is a nit, how about we make it
```scala
def as[A](implicit reader: ValueReader[A]): A = as(".")
```
The intention here is to expose the fact that if you pass in a "." as `path` you get the same result as `as`.